src/legacy/server/config/schema.js

```diff
@@ -135,7 +135,14 @@ export default () => Joi.object({
         then: Joi.default(!process.stdout.isTTY),
         otherwise: Joi.default(true)
       }),
-    timezone: Joi.string()
+    timezone: Joi.string(),
+    rotate: Joi.object().keys({
```

Comment:
This currently just takes an object, not a boolean.

Personal, I think we should drop supporting `rotate: false` and stick with `rotate.enabled`.

Reply:
that is what is happening already, however the documentation was not updated! I have done that, thanks for pointing it 👍